accord-core/src/main/java/accord/local/Status.java

```diff
@@ -913,7 +913,7 @@ public static Durability merge(Durability a, Durability b)
 
         public static Durability mergeAtLeast(Durability a, Durability b)
```

Comment:
rename or introduce `nonNullOrMergeAtLeast`?

Reply:
I think we do not need null checks there. Removed.